src/test/java/jenkins/plugins/hipchat/workflow/HipChatStepTest.java

```diff
@@ -0,0 +1,81 @@
+package jenkins.plugins.hipchat.workflow;
+
+import hudson.model.Result;
+import jenkins.plugins.hipchat.Messages;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.jenkinsci.plugins.workflow.steps.StepConfigTester;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runners.model.Statement;
+import org.jvnet.hudson.test.RestartableJenkinsRule;
+
+public class HipChatStepTest {
+    @Rule
+    public RestartableJenkinsRule story = new RestartableJenkinsRule();
+
+
+    @Test
+    public void configRoundTrip() throws Exception {
+        story.addStep(new Statement() {
+            @Override
+            public void evaluate() throws Throwable {
+                HipChatStep step1 = new HipChatStep("message");
+                step1.color = "green";
+                step1.room = "room";
+                step1.v2enabled = true;
+                step1.notify = false;
+
+                HipChatStep step2 = new StepConfigTester(story.j).configRoundTrip(step1);
+                story.j.assertEqualDataBoundBeans(step1, step2);
+            }
+        });
+    }
+
+    @Test
+    public void test_messasge() throws Exception {
```

Comment:
typo
These method names are not necessarily helpful (and are non camelCase).
Looking at these tests, I'm not sure I grasp their exact purpose, it just tests whether the step was invoked with null or non-null values, it doesn't actually test the fallback onto the global settings.